tests/no_create_accessanalyzer/main.tf

```diff
@@ -0,0 +1,14 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+module "create_iam_account" {
+  source = "../../"
+
+  providers = {
+    aws = aws
+  }
+
+  create_iam_account    = true
+  create_accessanalyzer = false
+}
```

Comment:
guess the eclint make target still needs some work..

Reply:
Yeah, idk what's going on with this. Regardless, I just updated Tim's PR and will be closing this one.